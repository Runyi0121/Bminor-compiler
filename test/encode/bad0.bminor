"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id libero ac odio vestibulum facilisis. Pellentesque in varius elit. Fusce nec mauris vel lectus malesuada consequat at quis ex. In interdum, justo nec volutpat ultrices, ante ligula fringilla dui, ac iaculis velit leo non urna. Duis ut nisi erat. Vivamus lacinia, risus vel tristique egestas, urna ipsum dictum sem, nec sagittis ipsum nisl eu odio. Fusce hendrerit tincidunt urna, eget fermentum ipsum tincidunt ac. Integer tincidunt magna ut ligula auctor, eu mattis mi ullamcorper. Nullam eget justo eget quam blandit mattis eu sit amet neque. Curabitur nec ullamcorper justo. Phasellus a tortor non nulla luctus volutpat eu in justo. Fusce euismod, augue a faucibus facilisis, quam velit bibendum tellus, vel suscipit justo dolor at lacus. Vivamus in nulla varius, non convallis odio."
