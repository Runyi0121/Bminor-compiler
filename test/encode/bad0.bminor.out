string exceeds the maximum length.
There is an error in the original string.
